src/main/java/com/tritoncubed/launchapi/Payload.java

```diff
@@ -31,50 +31,196 @@
     public void setUtc(Long utc) { this.utc = utc; }
 
     /**
-     * acceleration in the x direction (units) from accelerometer 
-     * Units
+     * acceleration in the x direction from accelerometer 
+     * Units is meters per second
+     * @param getAccel_X returns acceleration of x axis
+     * @param setAccel_X takes acceleration as arg and sets private var 
      */
-    private Short accel_X;
-    public Short getAccel_X() { return accel_X; }
-    public void setAccel_X(Short accel_X) { this.accel_X = accel_X; }
+    private float accel_X;
+    public float getAccel_X() { return accel_X; }
+    public void setAccel_X(float accel_X) { this.accel_X = accel_X; }
    
     /**
-     * accel_Y from accelerometer 
+     * acceleration in the y direction from accelerometer 
+     * Units is meters per second
+     * @param getAccel_Y returns acceleration of y axis
+     * @param setAccel_Y takes acceleration as arg and sets private var 
      */
-    private Short accel_Y;
-    public Short getAccel_Y() { return accel_Y; }
-    public void setAccel_Y(Short accel_Y) { this.accel_Y = accel_Y; }
+    private float accel_Y;
+    public float getAccel_Y() { return accel_Y; }
+    public void setAccel_Y(float accel_Y) { this.accel_Y = accel_Y; }
     
     /**
-     * accel_Z from accelerometer 
+     * acceleration in the z direction from accelerometer 
+     * Units is meters per second
+     * @param getAccel_Z returns acceleration of z axis
+     * @param setAccel_Z takes acceleration as arg and sets private var 
      */
-    private Short accel_Z;
-    public Short getAccel_Z() { return accel_Z; }
-    public void setAccel_Z(Short accel_Z) { this.accel_Z = accel_Z; }
+    private float accel_Z;
```

Comment:
Label the getters and setters independently 

Using document directives should follow this format: `@directive [label] <comment>`

Also, `@param` should be reserved for function parameters